Loop/View Controllers/CarbAbsorptionViewController.swift

```diff
@@ -504,19 +502,19 @@ final class CarbAbsorptionViewController: LoopChartsTableViewController, Identif
     
     @IBAction func presentCarbEntryScreen() {
         let navigationWrapper: UINavigationController
-        if closedLoopStatus.isClosedLoop {
+        if FeatureFlags.simpleBolusCalculatorEnabled && !closedLoopStatus.isClosedLoop {
```

Comment:
[nit] It seems non-obvious to me why the simple bolus calculator is only available when closed loop is false.  (Again, maybe it is just me).  Maybe consider a computed `var` that could be used both here (and above around line 67)?

Reply:
Hopefully my comment above about the rationale for Simple Bolus Calculator helps. If not, I'm open to suggestions for an intermediate var that you think would clarify things. What would you call it, and how would it be set?